app/lib/filters/radio_filter.rb

```diff
@@ -1,7 +1,36 @@
 module Filters
   class RadioFilter < Filter
     def value
-      parsed_value
+      return default_value unless acceptable_param?
+
+      return option_lookup_values(params) if multi_value?
+
+      Array(params)
+    end
+
+  private
+
+    def default_value
+      return option_lookup_values(default_allowed_value) if multi_value?
+
+      Array(default_allowed_value)
+    end
+
+    def option_lookup_values(val)
+      option_lookup.select { |key, _| val.include? key }.values.flatten
+    end
+
+    def default_allowed_value
+      @default_allowed_value ||= facet['allowed_values'].find(Proc.new { {} }) { |option| option['default'] }
```

Comment:
It sets a default return value of `{}` rather than the default `nil`. This is instead of doing `|| {}` afterwards. https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-find